### PR TITLE
MM T134 visual verification of tooltips

### PR DESCRIPTION
**Pull request by MikeworX**
**Summary**
This PR validates that several tool-tips appear as expected in the main view when the associated element is hovered over. The tool-tips checked are for the following elements:

- Member icon
- Plug-in icon (the demo plugin is used)
- Pinned posts - Recent Mentions
- Flagged posts as well as
- Add to Favorites
- Remove from Favorites
- Unmute (visible when channel is muted )
- 'Download' on download file icon
- Long channel name (shown truncated on the LHS)

The PR creates a new instance, along with a test user and a test channel with a long name, then proceeds to validate the hover tool-tips. The demo plugin is also installed and enabled for testing of the plug-in tool-tip, then removed during the after-hook.

Screen Shot 2020-10-01 at 1 51 40 PM

Ticket Links
[vercel](https://automation-test-cases.vercel.app/test/MM-T134)
[TM4J](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T134)
This is a re-do of the [original](https://github.com/mattermost/mattermost-webapp/pull/6573) pr which had significant merge issues.

### CODE REVIEW

**Review by saturninoabril — CHANGES_REQUESTED** [review]
@MikeworX Overall looks good, except for last couple comments.

File: e2e/cypress/integration/messaging/visual_verification_of_tooltips_on_top_nav,_channel_icons,_posts_spec.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+            // # Login as regular user
+            cy.apiLogin(user);
+
+            // # Set up test channel with a long name
+            cy.apiCreateChannel(testTeam.id, 'channel-test', 'Public channel with a long name');
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+    after(() => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/visual_verification_of_tooltips_on_top_nav,_channel_icons,_posts_spec.js:32]: Please separate with empty line for consistency.
```suggestion
    });

    after(() => {
```

**Review by saturninoabril — APPROVED** [review]
@MikeworX Thanks and looks good to me. Left last comment but approving now.

**Review by srkgupta — CHANGES_REQUESTED** [review]
Hi @MikeworX 

Overall the test looks good and is working fine. Just one suggestion:

The file name of the cypress test currently contains `,` which looks a little odd. Can you please get rid of this and rename the file.

**Review by srkgupta — APPROVED** [review]
Thanks @MikeworX. LGTM

**Pull request merged**